plugins/GSdx/GSCapture.cpp

```diff
@@ -525,7 +528,8 @@ std::wstring* GSCapture::BeginCapture(float fps, GSVector2i recommendedResolutio
 	}
 
 	m_capturing = true;
-	return new std::wstring();
+	filename = m_out_dir + "/audio_recording.wav";
+	return 1;
```

Comment:
I don't think I saw any other values besides 0 and 1 so maybe change the function to bool instead?